chapter_preliminaries/calculus_vn.md

```diff
@@ -524,6 +535,10 @@ for any $i = 1, 2, \ldots, n$.
 -->
 
 *dịch đoạn phía trên*
+1. Vẽ đồ thị của hàm số $y = f(x) = x^3 - \frac{1}{x}$ và đường tiếp tuyến của nó tại $x = 1$.
+1. Tìm gradient của hàm số $f(\mathbf{x}) = 3x_1^2 + 5e^{x_2}$.
+1. Gradient của hàm $f(\mathbf{x}) = \|\mathbf{x}\|_2$ là gì?
+1. Bạn có thể viết quy tắc dây chuyền cho trường hợp mà $u = f(x, y, z)$ ở đó $x = x(a, b)$, $y = y(a, b)$, và $z = z(a, b)$?
```

Comment:
Mình nhìn nhầm hay chữ chỗ này nhìn nghiêng nghiêng vậy?

Reply:
Nó nghiên do cái dấu `_` ở dòng trên trong latex mà markdown github nhận thành viết in nghiêng ấy. Khi render ra thì sẽ không bị.